src/main/webapp/WEB-INF/pages/developer/jqueryDev.jsp

```diff
@@ -0,0 +1,54 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Dinexpod
+  Date: 31.01.2019
+  Time: 20:04
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>jqueryDev</title>
+</head>
+<body>
+
+<button type="button" name="back" onclick="history.back()">back</button>
+
+<div id="allDevelopers">
+
+</div>
+
+
+<script
+        src="https://code.jquery.com/jquery-3.3.1.js"
```

Comment:
It's not good practice to use outer dependencies from web. What will you do if the servers of jquery.com will burn in flame? You can keep it in your project.